getting-started/default_posture.md

```diff
@@ -0,0 +1,149 @@
+# Default Posture
+
+KubeArmor supports configurable default mode of operations. Default Posture comes into picture when there's atleast one `Allow` policy i.e. KubeArmor is handling policies in whitelisting manner ( More about this in [Consideration in Policy Action](https://github.com/kubearmor/KubeArmor/blob/event-auditor/getting-started/consideration_in_policy_action.md) ).
+
+There are two default mode of operations available `block` and `audit`. `block` mode blocks all the operations that are not allowed in the policy. `audit` generates telemetry events for operations that would have been blocked otherwise.
```

Comment:
Shouldn't there be three options? allow/block/audit?

Allow mode would simply allow all the operations and only the explicitly denied operations would be blocked.

Reply:
> Allow mode would simply allow all the operations and only the explicitly denied operations would be blocked.

Why would one apply an 'Allow Policy' if this would be the case 😅 Won't this be the same as not applying any policy. Noting Default Posture only comes into picture when we have an Allow Policy.

Reply:
Sure, I see what your saying. But consider following scenario:

A user applies a block and an allow policy. Now the default posture could either be audit/deny. But just because user ended up adding an Allow policy, there is no way for the user to simply Allow everything else with getting alerts unless the user deletes the Allow policy. That is what I have issues with.

Reply:
Even in this case with the supposed allow mode set, What would be the expected behavior just of the Allow policy? From what I understand it would be same as removing it?

Reply:
Yes if the user removes the allow policy then it works fine. But asking user to remove a policy for something else to work was the point. Anyways, we can live with this behaviour. Thanks for the discussion.